File: Java/src/test/java/workshops/testingil/dirtytests/solutions/e10/builder/helpers/SequenceMessage.java
package workshops.testingil.dirtytests.solutions.e10.builder.helpers;

public class SequenceMessage {
    public String version;
    public char[] sequence;
    public boolean resetOnError;

    public void addSequence(String sequence) {
        this.sequence = sequence.toCharArray();
    }
}

File: C#/DirtyTestsWorkshop/Exercise/SequenceMessage.cs
namespace DirtyTestsWorkshop.Exercise
{
    public class SequenceMessage
    {
        public string version;

        public char[] sequence;
        public bool resetOnError;

        public void AddSequence(string sequence)
        {
            this.sequence = sequence.ToCharArray();
        }
    }
}

